solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestFeatureLtrScoringModel.java

```diff
@@ -46,8 +46,8 @@ public void getInstanceTest() throws FeatureException
   {
 
     final Map<String,Object> map = new HashMap<String,Object>();
-    map.put(ManagedFeatureStore.NAME_KEY, "test");
-    map.put(ManagedFeatureStore.CLASS_KEY, OriginalScoreFeature.class.getCanonicalName());
+    map.put("name", "test");
```

Comment:
So I'm not totally convinced that these variables shouldn't be public. I understand the desire for encapsulation, but feel in this case we are possibly hurting extensibility (same argument I made last time :P) .  However EVEN if we do make the assumption that these variables should be private in ManagedFeatureStore though, we should in my opinion at the bare minimum have static local variables within this package.